Jenkinsfile-e2e-local

```diff
@@ -18,24 +18,31 @@ node {
                     artifactDaysToKeepStr: '10'
                 )
             ),
-            pipelineTriggers([cron(cron_string)])
+            pipelineTriggers([cron(cron_string)]),
+            parameters(
+                [
+                    string(defaultValue: 'latest', name: 'BRANCH')
+                ]
+            )
         ]
     )
     timeout(time: 240, unit: 'MINUTES') {
         withEnv([
             'CI=true'
         ]) {
             cleanWs() // Clean the workspace
-            checkout scm // Checkout from git
 
-            docker.image("${nodeImage}").inside('--ipc host') {
-                try {
-                    Simorgh.installNodeModules()
-                    stage ('Localhost Tests') {
-                        Simorgh.runLocalAndProductionTests()
+            Simorgh.checkoutAndBuild("${params.BRANCH}")
```

Comment:
for test and live e2e's we do not need to build the application, so wastes a little time.

instead of `Simorgh.checkoutAndBuild` could we split out the checkout functionality from the build functionality.

`Simorgh.checkoutAndBuild`

could become 

```
checkout()
build()
```

with checkout being a new method.

Gotta love that Sudo code ;)